packages/ontario-frontend-cli/create/questions.js

```diff
@@ -50,13 +50,23 @@ const createQuestions = [
   },
   {
     type: 'list',
-    name: 'ESLint',
+    name: 'esLint',
     message: 'Want to add ESLint for fixing code problems?',
     choices: ['Yes', 'No'],
     default: 'No',
     filter: function(value) {
       return value === 'Yes' ? true : false;
     }
+  },
+  {
+    type: 'list',
+    name: 'prettier',
+    message: 'Want to add Prettier for code formatting?',
```

Comment:
I missed this wording in the other PR somehow, the change slipped past me. 

Can we make these messages more formal:
`Add and configure ESLint for finding and fixing code problems?`
`Add and configure Prettier for opinionated code formatting?`